www/js/services/unifiedDataLoader.ts

```diff
@@ -0,0 +1,68 @@
+import { getRawEntries } from './commHelper';
+import { ServerResponse, ServerData, TimeQuery } from '../types/serverData';
+
+/**
+ * removeDup is a helper function for combinedPromises
+ * @param list An array of values from a BEMUserCache promise
+ * @returns an array with duplicate values removed
+ */
+export const removeDup = function (list: Array<ServerData<any>>) {
```

Comment:
Note to self: we do the same deduplication here as we do in https://github.com/e-mission/e-mission-phone/pull/1086/files#diff-00cfb43594908885f4c2e9121a2224b2c080d9b96bff2e09ec6d4f8d8c6b0645R102-R104

Once both are merged it would be nice to use a common function in both places